decidim/decidim
# Add user moderation template

#### :tophat: What? Why?
This PR adds a new admin section in the templates area, to allow admins to fill in moderation templates, that can be used later. It automatically supports multiple translations, rendering back to the interface the template in admin's language (current_locale of current logged in user).
Please refer to:  #10033 for further details. 

This addresses SPAM01 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to  #10015
- Fixes  #10033

#### Testing
1. go to admin / templates/  Block User Messages 
2. Add a simple template 
3. Go to frontend, find an user, and report it 
4. Visit the admin / Participants / Reported users 
5. Click "Block user" icon 
6. See new Choose template
7. Choose the previously added template 
8. Observe 

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Empty templates list](https://user-images.githubusercontent.com/105683/199347614-b0dbf0ac-aa99-4acf-8023-36d3042a19dd.png)
![Adding new template](https://user-images.githubusercontent.com/105683/199347834-16a721c8-38a6-4853-84b0-a6f52e01f51d.png)
![Listing templates](https://user-images.githubusercontent.com/105683/199347902-12374faa-6f2f-4583-b7e3-aed9bd159211.png)
![Blocking an user](https://user-images.githubusercontent.com/105683/199348003-e5558d08-876a-4bdb-b618-96f819413473.png)
![Select template](https://user-images.githubusercontent.com/105683/199348070-ba5b3367-4fcb-4ddb-9ddf-8f25fc51a94e.png)

:hearts: Thank you!
